### ISSUE TITLE
[Telegram Chat Bot] Users are unable to navigate through the main menu using the Tab key in the UbsSenderBot

### ISSUE THREAD

**Issue by karinayakovenko** [ita-social-projects/GreenCity#9244]
**Environment:** Windows 11 Home, Telegram desktop app
**Reproducible:** always
**Build found:** 26.10.2025
**Preconditions**
- Telegram is installed
- UbsSenderBot is opened (/start)
- The main menu with all options is displayed

**Steps to reproduce**
1. Use the Tab key on the keyboard to navigate between the main menu buttons

**Actual result**
Tab key navigation for selecting and switching main menu options is currently not functioning as expected.

**Expected result**
User can select and switch between main menu items using the Tab key.

**User story and test case links**
 User story #9046
